src/core/Akka.Remote.TestKit/MultiNodeSpec.cs

```diff
@@ -36,6 +36,9 @@ public abstract class MultiNodeConfig
         // it will remain safe by defaut
         Config _commonConf = ConfigurationFactory.Empty;
 
+        //makes log level configurable
+        public string LogLevel = "DEBUG";
```

Comment:
Do you think we need to modify the spec definition/configuration? Or probably we can keep specs untouched, and just apply logs filtering on MultiNodeTestRunner side?

Reply:
When LogLevel is supplied vai command line, it is stored here.
````csharp
LogLevel = CommandLine.GetPropertyOrDefault("multinode.log.level", "DEBUG");
            
````